playbooks/de-populate-consul.yaml

```diff
@@ -0,0 +1,26 @@
+---
+
+- name: "Update keys in consul"
```

Comment:
(opinion point: should this playbook delete everything under `configs/{{environment_name}}` before updating things, or should that be a.) somewhere else, b.) not worried about?)